core/commands/ping.go

```diff
@@ -131,14 +132,18 @@ func pingPeer(ctx context.Context, n *core.IpfsNode, pid peer.ID, numPings int)
 			n.Peerstore.AddAddrs(p.ID, p.Addrs, pstore.TempAddrTTL)
 		}
 
-		outChan <- &PingResult{Text: fmt.Sprintf("PING %s.", pid.Pretty())}
+		outChan <- &PingResult{
+			Text:    fmt.Sprintf("PING %s.", pid.Pretty()),
+			Success: true,
+		}
 
 		ctx, cancel := context.WithTimeout(ctx, kPingTimeout*time.Duration(numPings))
 		defer cancel()
 		pings, err := n.Ping.Ping(ctx, pid)
 		if err != nil {
 			log.Debugf("Ping error: %s", err)
-			outChan <- &PingResult{Text: fmt.Sprintf("Ping error: %s", err)}
+			outChan <- &PingResult{
+				Text: fmt.Sprintf("Ping error: %s", err)}
```

Comment:
lets explicitly set Success false here (for clarity to the reader), and then put the closing bracket on its own line.